src/main/java/org/opensrp/web/rest/RestUtils.java

```diff
@@ -92,48 +106,50 @@ public static boolean getBooleanFilter(String filter, HttpServletRequest req) {
 	public static void main(String[] args) {
 		System.out.println(new DateTime("​1458932400000"));

```

Comment:
Unrelated to this update, but should we still have this?

Reply:
I am not sure. The code works without that main method